apps/web/app/(app)/gardens/[chain]/[garden]/pool/[poolId]/page.tsx

```diff
@@ -69,14 +92,16 @@ export default async function Pool({
           {/* Description section */}
           <section className="relative flex w-full flex-col items-center overflow-hidden rounded-lg border-2 border-black bg-white">
             <div className="mt-4 flex w-full flex-col items-center gap-12 p-8">
-              <h3 className="max-w-2xl  text-center font-semibold">{title}</h3>
+              <h3 className="max-w-2xl  text-center font-semibold">
+                {ipfsResult?.title}
+              </h3>
               {!isEnabled && (
                 <div className="badge badge-warning absolute left-5 top-5 gap-2 p-4 font-bold">
                   Pending review from community council
                 </div>
               )}
 
-              <p>{description}</p>
+              <p>{ipfsResult?.description}</p>
```

Comment:
should we have spinner here too?

Reply:
https://github.com/1Hive/gardens-v2/pull/229/commits/62f3fa611d9e6d3642dc52e515e8c95100005af3